src/main/java/com/zendesk/maxwell/MaxwellParser.java

```diff
@@ -92,17 +92,23 @@ public void run() throws Exception {
 			if ( event == null )
 				continue;
 
-			producer.push(event);
+			if ( !skipEvent(event)) {
+				producer.push(event);
 
-			// TODO:  this isn't quite right: we need to only stop on table map events,
-			// although it's unclear to me whether you can have two row events in a row.
-			// I think maybe you can.
+				// TODO:  this isn't quite right: we need to only stop on table map events,
+				// although it's unclear to me whether you can have two row events in a row.
+				// I think maybe you can.
 
-			BinlogPosition p = eventBinlogPosition(event);
-			this.config.setInitialPosition(p);
+				BinlogPosition p = eventBinlogPosition(event);
+				this.config.setInitialPosition(p);
+			}
 		}
 	}
 
+	private boolean skipEvent(MaxwellAbstractRowsEvent event) {
+		return event.getTable().getDatabase().getName().equals("maxwell");
+	}
```

Comment:
If you think about this a bit, this is fine. 

Migration -> maxwell writes -> kafka push. (so if we follow maxwell, we get the schema changes as snapshots)

But I think it's best to not complicate at the beginning :)